py3-certipy.yaml

```diff
@@ -0,0 +1,36 @@
+# Generated from https://pypi.org/project/certipy/
+package:
+  name: py3-certipy
+  version: 0.1.3
+  epoch: 0
+  description: Utility to create and sign CAs and certificates
+  copyright:
+    - license: BSD
+  dependencies:
+    runtime:
+      - py3-pyopenssl
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 695704b7716b033375c9a1324d0d30f27110a28895c40151a90ec07ff1032859
+      uri: https://files.pythonhosted.org/packages/3a/b3/f9228354c1eac06cd3577a981571b9188392d73d583fcaa7a8f3fb374a56/certipy-${{package.version}}.tar.gz
+
+  - name: Python Build
+    uses: python/build-wheel
+
+  - uses: strip
+
+update:
+  enabled: false # doesn't exist in ReleaseMonitor, no releases or tags in GitHub, no new release since 2019.
```

Comment:
Are we sure this is not maintained? I did come across this Git repo, but i'm not sure if its the same application:
- https://github.com/ly4k/Certipy

Reply:
Yep, the corresponding repo is: https://github.com/LLNL/certipy